index.html

```diff
@@ -661,145 +696,207 @@ <h2>Data Model and Representations</h2>
-        Specification Registries</a>.
-      </p>
+        <p>
+          For example, if a producer adds an `@context` entry to a DID document
+          in the `application/did+json`
+          representation, some consumers (those that understand JSON-LD, such as
+          JSON-LD document loaders) may process
+          it
+          in one way, while other consumers (those that only understand plain
+          JSON, without any JSON-LD specific rules)
+          may process it in a different way.
+        </p>
+
+        <p>
+          This can lead to inconsistent behavior by consumers that cannot be
+          predicted by a producer.
+        </p>
+
+      </section>
+
+      <section class="informative">
+        <h3>
+          Reliance on representation-foreign entries
+        </h3>
+
+        <p class="advisement">
+          It is bad practice and potentially harmful for consumers to rely on
+          "representation-foreign entries" in DID
+          documents.
+        </p>
+
+        <p>
+          As a special case of the previous section, certain consumers may go as
+          far as relying on the presence of
+          representation-foreign entries. For example, an incorrectly
+          implemented consumer of DID documents in the
+          `application/did+json` representation may attempt to apply JSON-LD
+          tooling to that representation. That
+          tooling
+          is dependent on the presence of the `@context` representation-foreign
+          entry, which is not actually required
+          (and
+          in fact discouraged) to be used in the `application/did+json`
+          representation.
+        </p>
+
+        <p>
+          The result is lack of interoperability, since such consumers will
+          throw errors even if the returned DID
+          document
+          representation is actually conformant.
+        </p>
+
+      </section>
+
+      <section class="informative">
+        <h3>
+          Security problems with representation-foreign entries
+        </h3>
+
+        <p>
+          If some implementations of DID document consumers process
+          representation-foreign entries, and others don’t,
+          then
+          this can lead to security holes, since the semantics of DID document
+          properties will not be interpreted in a
+          predictable way.
+        </p>
 
-      <pre class="example nohighlight" title="A DID document in the `application/did+json` representation with the `@context` representation-foreign entry">
+        <p>
+          For example, consider the following DID document, and assume that the
+          JSON-LD context
+          `https://myextensions.com/other-definitions.jsonld` defines
+          `blockchainAccountId` in a way that is different
+          from the <a
+            href="https://w3c.github.io/did-spec-registries/#blockchainaccountid">property
+            listed in the DID
```

Comment:
This should be changed to `https://www.w3.org/TR/did-spec-registries/#blockchainaccountid` (or its `data-cite` equivalent). It is not appropriate to refer to the editor's draft if there is an official /TR document.